lib/findingaids/ead/indexer.rb

```diff
@@ -49,11 +49,18 @@ def reindex_changed_since_yesterday
     reindex_changed(commits('--since=1.day'))
   end
 
-  # Reindex all files changed in the last day
+  # Reindex all files changed in the last week
   def reindex_changed_since_last_week
     reindex_changed(commits('--since=1.week'))
   end
 
+  # Reindex all files changed since x days ago
+  def reindex_changed_since_x_days_ago(x)
+    # assert that argument can be converted to an integer
+    days = Integer(x)
```

Comment:
@jgpawletko You could require an integer instead:

```
raise ArgumentError.new("Expecting #{x} to be an integer") unless x.is_a? Integer
```

Also for readability maybe keep the 'x' which is meant as the parameter out of the function name:

`reindex_changed_since_days_ago(days_ago)`

Do you want to just allow a `days_ago` param or should it also be configurable by date? In that case there could be two wrapper functions, the one above and `reindex_changed_since_date(date)` where date is a `Date` string.